proto/viam/app/v1/app.proto

```diff
@@ -541,6 +541,8 @@ message Fragment {
   int32 robot_part_count = 9;
   // number of organizations using this fragment
   int32 organization_count = 10;
+  // whether the organization(s) using this fragment is the same as the fragment org
+  bool is_only_used_by_owner = 11;
```

Comment:
To match convention, I would change to `only_used_by_owner`. The rest of the boolean fields do not have `is_` prefix.

Reply:
Thanks sounds good!